### PR TITLE
Try to simplify further — put all rendering logic in one place — and show first three sentences of fact set company descriptions

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

**Comment by matthew-andrews** [comment]
@i-like-robots

File: server/controllers/stream.js [hunk 1]
```diff
@@ -47,84 +53,112 @@ module.exports = function(req, res, next) {
...
+
+			if (stream.taxonomy === 'organisations') {
+				promises.push(api.mapping(stream.tag.term.id, stream.tag.term.taxonomy)
+					.then(function(mapping) {
+						return api.organisations({ uuid: mapping.apiUrl.replace(/^http:\/\/api\.ft\.com\/organisations\//, '') })
+					})
+
+					// HACK: Show the first three sentences
+					.then(function(organisation) {
+						var profile = organisation.profile.replace(/^<\/?body>/g, '')
```
Comment (i-like-robots) [inline server/controllers/stream.js]: > 😢
- JSHint

File: server/controllers/stream.js [hunk 2]
```diff
@@ -47,84 +53,112 @@ module.exports = function(req, res, next) {
...
+		.then(function(stream) {
+			var promises = [];
 
-				res.render('layout', stream);
+			if (isTaxonomySearch && res.locals.flags.dynamicTertiaryNav) {
+				promises.push(relatedTags(stream.taxonomy + ':' + stream.title)
+					.then(function(tags) { stream.dynamicTertiaryNav = tags.slice(0, 5); }));
+			}
+
+			if (stream.taxonomy === 'organisations') {
```
Comment (i-like-robots) [inline server/controllers/stream.js]: Is any of this needed if a user requests RSS?

File: server/controllers/stream.js (hunk 1 above)
Comment (matthew-andrews) [inline server/controllers/stream.js]: how come?

Comment (i-like-robots) [inline server/controllers/stream.js]: Lack of semi-colon :eyes:

Comment (matthew-andrews) [inline server/controllers/stream.js]: oh

Comment (i-like-robots) [inline server/controllers/stream.js]: At some point I'll stop being cryptic, I promise.

**Pull request merged**